src/components/ActionsBuilder/SupportedActions/SetGuildConfig/SetGuildConfigEditor.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { render } from 'utils/tests';
+import { BigNumber } from 'ethers';
+import SetGuildConfigEditor from './SetGuildConfigEditor';
+import { emptyDecodedCall } from './fixtures';
+import { FIELDS } from './constants';
+
+const mockBigNumber = BigNumber.from(100000000);
+
+jest.mock('Modules/Guilds/Hooks/useGuildConfig', () => ({
+  useGuildConfig: () => ({
+    data: {
+      permissionRegistry: '0x23120390909dq0w9adsj832921293989',
+      name: 'TEST GUILD',
+      proposalTime: mockBigNumber,
+      timeForExecution: mockBigNumber,
+      maxActiveProposals: mockBigNumber,
+      votingPowerForProposalCreation: mockBigNumber,
+      tokenVault: '0X9628SD7128DAS78SHDF8HF832H283FH283RH98ASHD',
+      lockTime: mockBigNumber,
+      voteGas: mockBigNumber,
+      maxGasPrice: mockBigNumber,
+      votingPowerPercentageForProposalExecution: mockBigNumber,
+      votingPowerPercentageForProposalCreation: mockBigNumber,
+      minimumMembersForProposalCreation: mockBigNumber,
+      minimumTokensLockedForProposalCreation: mockBigNumber,
+      votingPowerForProposalExecution: mockBigNumber,
+      token: '0x30ii2jnn32ru8238r293ru2389ru2938ru239r23r23',
+    },

```

Comment:
Should we move this to the `fixtures` file?

Reply:
I don't think we need to use fixtures for mocks. Fixtures are more for testing diff outcomes I guess. Also, jest doesn't like out-of-scope content for some reason.